src/pages/studyView/UserSelections.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { observer } from "mobx-react";
+import { computed } from 'mobx';
+import styles from "./styles.module.scss";
+import { StudyViewFilter } from 'shared/api/generated/CBioPortalAPIInternal';
+import { ChartMeta } from 'pages/studyView/StudyViewPageStore';
+
+export interface IUserSelectionsProps {
+    filter: StudyViewFilter;
+    attributesMetaSet: { [id: string]: ChartMeta };
+    updateClinicalDataEqualityFilter: (chartMeta: ChartMeta, value: string[]) => void;
+    clearGeneFilter: () => void;
+    clearCNAGeneFilter: () => void;
+    clearCustomCasesFilter: () => void;
+    clearAllFilters: () => void
+}
+
+
+@observer
+export default class UserSelections extends React.Component<IUserSelectionsProps, {}> {
+
+    constructor(props: IUserSelectionsProps) {
+        super(props);
+    }
+
+    @computed get showFilters() {
+        return !_.isEmpty(this.props.filter) && (
```

Comment:
Wondering whether this should be a util function? This could be used in some other places too? Can't think about a use case though.

Reply:
@kalletlak I also noticed a similar if condition in the cache, but actually it is not logically equivalent to this one. Is this is intentional or is it a bug? 

https://github.com/cBioPortal/cbioportal-frontend/blob/090546ff443214e716ab484273beca20dd625d4f/src/shared/cache/StudyViewClinicalDataCountsCache.ts#L45

At some point, we will also need to add `clinicalDataIntervalFilters` (which will be introduced by bar chart API) both here and in the cache.

Reply:
It looks like same. I'll move it utils